be/src/exec/csv_scanner.cpp

```diff
@@ -506,4 +514,45 @@ void CSVScanner::_report_rejected_record(const CSVReader::Record& record, const
     _state->append_rejected_record_to_file(record.to_string(), err_msg, _curr_reader->filename());
 }
 
+static TypeDescriptor get_type_desc(const Slice& field) {
+    StringParser::ParseResult result;
+
+    StringParser::string_to_int<int64_t>(field.get_data(), field.get_size(), &result);
```

Comment:
Will it return SUCCESS to parse `123.4` to an int64_t?

Reply:
No.